models/action.go

```diff
@@ -283,11 +283,12 @@ func (a *Action) GetIssueContent() string {
 
 // GetFeedsOptions options for retrieving feeds
 type GetFeedsOptions struct {
-	RequestedUser   *User // the user we want activity for
-	Actor           *User // the user viewing the activity
-	IncludePrivate  bool  // include private actions
-	OnlyPerformedBy bool  // only actions performed by requested user
-	IncludeDeleted  bool  // include deleted actions
+	RequestedUser   *User  // the user we want activity for
+	Actor           *User  // the user viewing the activity
+	IncludePrivate  bool   // include private actions
+	OnlyPerformedBy bool   // only actions performed by requested user
+	IncludeDeleted  bool   // include deleted actions
+	Date            string // the day we want activity for: YYYY-M-D
```

Comment:
Would you mind changing the signature of `GetFeedsOptions` to not specify a single date, but to contain a time range (i.e. `after, before time.Time`, populate these values in `home.go`/`profile.go`)? In #14080 I'm making this struct reusable for the heatmap, and a time-range filter would be welcome for the heatmap too (for #5439).